roles/elasticsearch/vars/main.yml

```diff
@@ -1,2 +1,3 @@
 ---
 # vars file for elasticsearch
+elasticsearch_major_version: "{{ '8.x' if acs.version is version('25.0', 'ge') else '7.x' }}"
```

Comment:
the elasticsearch role should not know anything about acs. Please move that logic to the playbook